### PR TITLE
Improvement: menu icons

### LINKED ISSUE #3251: Buttons for user queries and oldest/newest are the same
Hello everyone, just a minor issue which didn't bother me on PC screen but started to confuse me on a mobile screen:

Sans titre

The buttons are the same! Maybe we could find something else for one of them?

### PR DESCRIPTION
**Pull request by math-GH**
Closes #3251
Address partially #3380

(this PR finishes  #3471, that I could not edit itself so I copied it and made the next steps )

Before:
grafik
Hover:
grafik

Theme Swage:
grafik

After;
grafik
Hover:
grafik

Thee Swage:
grafik

Changes proposed in this pull request:

- Add title on user query menu item (For #3380)
- new icon for the sorting button
- new icon for the User Query button (used the icon from the User Query menu and deleted the plus symbol)
- some themes got an improvement while hovering over the "Bookmark current query"

How to test the feature manually:

- Hover the user query drop-down menu to see the tooltip

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

**Comment by aledeg** [comment]
I guess I'll close #3471. Thank you @math-GH for taking over!

File: p/themes/Flat/icons/bookmark-tag.svg [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   id="svg8"
+   sodipodi:docname="bookmark-tag.svg"
+   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20)"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg">
+  <defs
+     id="defs12" />
+  <sodipodi:namedview
```
Comment (Alkarex) [inline p/themes/Flat/icons/bookmark-tag.svg]: Can the SVG be exported without the proprietary extensions? Would we miss anything important for further edition (including the case of other editors)?

Comment (math-GH) [reply p/themes/Flat/icons/bookmark-tag.svg]: done

**Comment by math-GH** [comment]
Made the compression with https://jakearchibald.github.io/svgomg/
If you like it, than I would compress the other icons too (in another PR). Could save in average 15% of file size.

**Comment by Alkarex** [comment]
The new SVGs look good, @math-GH 
As a personal preference, I like introducing new lines to avoid a single line that is much too long (problematic for editors, diffs, manual checks...)

**Review by Alkarex — APPROVED** [review]

**Comment by Alkarex** [comment]
P.S. You are welcome to give a shot at the other SVGs, but we need to check in a few different browsers and platforms for possible rendering regressions

**Comment by Alkarex** [comment]
I like it, except a little minus on the icon to open the user queries, as it is now not clear that it can open a menu.
Improvement on that specific point welcome, otherwise I can live with it

File: p/themes/icons/bookmark-add.svg [hunk 2]
```diff
@@ -1,6 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" height="16" width="16">
-<g transform="translate(-141.0002,-807)" fill="#bebebe">
-<path d="m143,807,0,13,4-4,4,4,0-4,0-1-2,0,0-4,2,0,0-4z"/>
-<path d="m152,810,0,2-2,0,0,2,2,0,0,2,2,0,0-2,2,0,0-2-2,0,0-2-2,0z"/>
-</g>
-</svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><g fill="#666"><path d="M1.5 1.5v13l4-4 4 4V10H7V5h2.5V1.5Z"/><path d="M10.5 4.5v2h-2v2h2v2h2v-2h2v-2h-2v-2z"/></g></svg>
```
Comment (Frenzie) [inline p/themes/icons/bookmark-add.svg]: I'd also prefer to maintain the newlines so it's clearer what's different and what's not.

Btw, you have to be very careful that such path simplifications don't lose info that was there in the path previously (although in this case I think the simplification is likely mainly related to that transform, but I haven't checked it visually). Smoothing out some features of the path can even be desirable, mind you.

Comment (math-GH) [reply p/themes/icons/bookmark-add.svg]: see: https://github.com/jakearchibald/svgomg/issues/375

Comment (math-GH) [reply p/themes/icons/bookmark-add.svg]: The files are now formated

**Comment by math-GH** [comment]
> I like it, except a little minus on the icon to open the user queries, as it is now not clear that it can open a menu. Improvement on that specific point welcome, otherwise I can live with it

I thought about this "issue" too. But, the config button has not a "dropdown" logo inside. That is why I thought it could be ok,

File: p/themes/icons/bookmark-add.svg [hunk 3]
```diff
@@ -1,6 +1,6 @@
-<svg xmlns="http://www.w3.org/2000/svg" height="16" width="16">
-<g transform="translate(-141.0002,-807)" fill="#bebebe">
-<path d="m143,807,0,13,4-4,4,4,0-4,0-1-2,0,0-4,2,0,0-4z"/>
-<path d="m152,810,0,2-2,0,0,2,2,0,0,2,2,0,0-2,2,0,0-2-2,0,0-2-2,0z"/>
-</g>
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16">
+	<g fill="#666">
```
Comment (Frenzie) [inline p/themes/icons/bookmark-add.svg:2]: The indentation's very much optional to me though, unless it were really complex maybe. :-)

(If you and/or @Alkarex prefer it this way it's certainly fine by me.)

**Comment by math-GH** [comment]
> P.S. You are welcome to give a shot at the other SVGs, but we need to check in a few different browsers and platforms for possible rendering regressions

Checked on Windows: Firefox, Edge, Chromium.

**Pull request merged**